public/css/cclw/theme.css

```diff
@@ -0,0 +1,83 @@
+#cclw {
+  --text-h1: 48px;
+  --text-h2: 36px;
+  --text-h3: 20px;
+  --text-h4: 21px;
+  --text-h5: 18px;
+  --text-h6: 16px;
+
+  --font-weight-medium: 540;
+
+  /* @layer base {
+    h1 {
+      font-size: var(--text-2xl);
+    }
+    h2 {
+      font-size: var(--text-xl);
+    }
+  } */
+}
+
+/*
+  TODO: remove specific css styles and overrides by moving into their components
```

Comment:
Happy with this approach for now.

Reply:
Please see the `/src/styles/cclw` for the correct implementation.

This approach did not work because in the public directory I do not have access to tailwind.